README.md

```diff
@@ -256,12 +256,25 @@ To learn about how to use Porcupine with Android Speech to Text recognition go t
 
 ### iOS Demos
 
-Before building the demo app, run the following from this directory to install the Porcupine-iOS Cocoapod:
-```ruby
+#### BackgroundService Demo
```

Comment:
Maybe have a quick sentence above explaining the difference between the two demos